druidfi/stonehenge
# Refactor

Tested with
- macOS Catalina
- Ubuntu 18.04 LTS
- Manjaro 18 (Arch)

New features:
- mkcert installation by default and using HTTPS
- pre and post actions for up
- fixed version for Traefik 1 (now that 2 is released)
- removed need for external bash scripts
- custom service-not-found page

How to test:
- Stop Stonehenge if running: `make down`
- Clone the repo if you don't have it
- Checkout this branch `git checkout refactor`
- Check debug with `make debug`
- Check ping with `make ping`
- Start Stonehenge with `make up`
- Can you access https://portainer.docker.sh without problems?
